sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,174 @@
+- Disable parallel for range rounding at -O0 [2c117d7a61041fcb06c0e42715fcc81459e4bb8c]
+- Enable SPV_INTEL_optnone extension for non-fpga targets [993fe5b31b43911a903463e414f1daf1ab81337c]
+- Relax LowerESIMDSlmReservation assert [7897517c3a5ebbe4e05977d9b8cc0dd28a192d98]
+
+### SYCL Library
+- vec abi unification and trivially copyable [531aabfdee91e0055e3d28459432dfa59b17850d] [f4ed132f243ab43816ebe826669d978139964df2]
+- Support missing types with ldg extension [aec8a35788c1953c6ae74ab1e6eb924fa09d5e4e]
+- Stop throwing exception when passing empty accessor to handler::require() [7bf1f57bc26e11868d2ef136b6e5ab18ea5d519c]
+- Move addc() and subb() functions out of experimental namespace [9e3852bc1c2e52e572a7617f924d548f80300222]
+- Move BFN function from experimental namespace [c9da5c7ff7e80b193491932ebd9c66ee0777d1f2]
+- Report source code info(file/line number) where null pointer exception occurred in memcpy [626c4b5078c14e90a66a692ac306d98065ece035]
+- Improve accuracy of double tanpi host implementation [041a74c0cbda14fc8e03e3944367a8b0107656b6]
+- Add more support for AMD subgroup functions [288aeaef03a9100bfd927aab641730da6832241d]
+- annotated_ptr API fixes [954730e7a8232087f9bca19b26a7c2f8d5bf08f3] [fbad42a398efca3aa04bdba6504885abc9b333f7] [43336a15fcafec1b3c3e94753023304ec6ef0ed3]
+- Implement fallback path for profiling info support [7cfa951303eb828836637c21518ed0f1d27ad753]
+- Fix require alignment in buffers [093dae1f72146ef0f640d2e667576811245eb078]
+- Throw an exception when unsupported features used in a graph [8d8d3f4f52b0c7710bafdb6acd7d9157c6a0b7d4]
+- Support get_multi_ptr in ESIMD context [2f0f3814c5415068ad77ae954580b4a7618cccdd]
+- Accept integral type in parallel_for with offsets [ecfc49c825c665047ada8be42ccb3202e4111227]
+- Support assert() in ESIMD kernels [545a043ebaf52976b4a3c7bf35e2ea637428c505]
+- Add missing Intel archs to sycl_ext_oneapi_device_architecture [c4db251179dcd1b30a1043d4780634dbe39b3e3a]
+- Enable range rounding for unnamed lambdas [4f30e6648d9e38fefcc1f606638ea6d6604390bb]
+- Replaced LoadLibraryA() with LoadLibraryEx() [713442327d4038d5cf2158c1d3d85920a30d61cf]
+- Don't use legacy ANSI-only Windows API for loading plugins [5c30815d48211565812adb68f95d9b6fb788a39e]
+- Use SFINAE to reject wrong signature of CGF object parameter [5801970585b2724dae8367470152304cd2718e3c]
+- fill() use native functions instead of custom kernels [0249a55f810eedc42c3571cff176c5ab01606793]
+- Add defaults for streaming and register_map fpga_kernel_properties [23a6f389c1e45df077c6f15b691835b2976fda4d]
+- Move GRF property conflict check to compile time [4fb92d35c7beac85913edd23197c2e6506e49240]
+- Add error when using invoke_simd with callables with struct arguments [03c13fd53a2413de41b68fd03d7ad7e7ef905215]
+- Add missing rintf [3c327c736ced3690b283f04302146ac952f53726]
+- Cache the max local mem size in CUDA adapter [1c892f952cae7f0cea5c446534441478948e3f7d]
```

Comment:
For users, this should probably be rewritten as "optimized something..something". Tagging @hdelan

Reply:
I went with "Optimize kernel launch on CUDA by caching the max local mem size so that CUDA driver is called less"